lib/model/game_state.dart

```diff
@@ -50,10 +51,17 @@ class GameState with ChangeNotifier {
     randomiser.shuffle(deck);
   }
 
+  void viewDiscards() {
```

Comment:
However this ends up looking, we should add tests for it. Feels a bit silly, especially if all it ends up doing is setting a flag, but meh.

Thinking that we might actually end up with this being `toggleDiscardView()`, which would just negate `viewingDiscard`. This way we can hook back into it from an escape button in our discard view (I'm guessing that currently there's no way to get out of this view once you're in it?)

Also, I'm guessing that currently we don't have a way to escape from this view once we're in it?